packages/@adobe/spectrum-css-temp/components/sidenav/index.css

```diff
@@ -21,8 +21,6 @@ governing permissions and limitations under the License.
   margin: 0;
   padding: 0;
   outline: none;
-
-  height: 100%;
```

Comment:
So whats going on here? Why do we need to change the Spectrum css?
@snowystinger

Reply:
ah, yep, i meant in the docs css

Reply:
thanks @dannify + @snowystinger -- updated.